hooks/useLoadData/useLoadData.ts

```diff
@@ -186,7 +194,8 @@ export function useLoadData<T extends NotUndefined, Deps extends any[]>(
     }
   }, [counter, localFetchWhenDepsChange]);
 
-  const nonPromiseResult = initialPromise.res instanceof Promise ? undefined : initialPromise.res;
+  const initialPromiseRes = initialPromise.res;
+  const nonPromiseResult = isPromise(initialPromiseRes) ? undefined : initialPromiseRes;
```

Comment:
I would probably eliminate line 197 and just do:

```suggestion
  const nonPromiseResult = isPromise(initialPromise.res) ? undefined : initialPromise.res;
```